compiler/src/dotty/tools/dotc/ast/Positioned.scala

```diff
@@ -24,13 +24,7 @@ abstract class Positioned(implicit @constructorOnly src: SourceFile) extends Pro
   def uniqueId: Int = myUniqueId
 
   def uniqueId_=(id: Int): Unit = {
-    if (Positioned.debugId == id) {
-      def printTrace() = {
-        val stack = Thread.currentThread().getStackTrace().map(">   " + _)
-        System.err.println(stack.mkString(s"> Debug tree (id=${Positioned.debugId}) creation \n> $this\n", "\n", "\n"))
-      }
-      printTrace()
-    }
+    if (Positioned.debugId == id) ???
```

Comment:
No, printing the tree and the stack without stopping has been proven to be extremely useful for position debugging.

Maybe we should add a `-Ychrash-tree-with-id` that extends this functionality.

Reply:
Or we can ask for a prompt to continue or not, as `-Xprompt`?